config/migrations/2023/20230928110550-add-privacy-metadata.sparql

```diff
@@ -0,0 +1,40 @@
+
```

Comment:
I think it's easier to get rid of this migration and use a convention to link the privacy graph to an organization graph. Otherwise you'll also need to add this triple in the OP consumer every time a new bestuur is created

Reply:
the convention chosen is:
`http://mu.semte.ch/graphs/organizations/{bestuurUuid}/privacy`